logs/logs.go

```diff
@@ -0,0 +1,65 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
```

Comment:
It does make sense for auth.go to store the parsed token in the request context so that other users of the token do not need to parse/decrypt the token again.  To make that change you should use the context pkg from `http://www.gorillatoolkit.org/pkg/context`.  So auth.go will store it in the context and this code will pull from the context.